### DISCUSSION TITLE
Human error EventTarget in NodeJS 14.7

### DISCUSSION THREAD

**Discussion by olmerg** [vladmandic/human#174] [Q&A] [answered]
amazing library, thanks in advance by your help.

After I install Human in node v14.7.6 /Ubuntu 20.04 and try to create a Human object inside my node project I receive and error in line 12580 of human.node.js `this.events = new EventTarget();`

If I comment this line in the library it works, what else should I install?

**Comment by vladmandic** [comment, marked as answer]
I've created an issue #175 to track this.
